authorize_request_handler.go

```diff
@@ -275,7 +281,51 @@ func (f *Fosite) validateResponseMode(r *http.Request, request *AuthorizeRequest
 	return nil
 }
 
+func (f *Fosite) authorizeRequestFromPAR(ctx context.Context, r *http.Request, request *AuthorizeRequest) (bool, error) {
+	requestURI := r.Form.Get("request_uri")
+	if requestURI == "" || !strings.HasPrefix(requestURI, f.PushedAuthorizationRequestURIPrefix) {
+		// nothing to do here
+		return false, nil
+	}
+
+	clientID := r.Form.Get("client_id")
+
+	storage, ok := f.Store.(PARStorage)
+	if !ok {
```

Comment:
We have a similar error check in the code above. Can we introduce a singular error that we reuse when the storage does not implement PAR?

Reply:
@vivshankar did you have time to address this comment?

Reply:
I have addressed this comment by adding a specific error message for this. Not quite sure what you mean by "similar error check".

Reply:
We check on multiple occasions that the config or store implement PAR. We could use a single error `var PARNotImplemented = ...` and reuse that error instead of writing new errors for every check